src/test/java/org/minions/devfund/mauricioz/linkedList/ContainsTest.java

```diff
@@ -0,0 +1,366 @@
+            Assert.assertTrue("containsSubsequence returns false when first LinkedList contains second at "
+                    + "rear of list", var3);
+        } catch (Exception var4) {
+            Assert.fail("containsSubsequence throws " + var4 + " when first LinkedList contains second "
+                    + "at rear of list");
+        }
+
+    }
+
+    /**
+     * test.
+     */
+    @Test
+    public void testNoOverlap() {
+        LinkedList var1 = new LinkedList();
+        LinkedList var2 = new LinkedList();
+        var1.add(LIST1FIRSTVALUE);
+        var1.addLast(LIST1SECONDVALUE);
+        var1.addLast(LIST1THIRDVALUE);
+        var1.addLast(LIST1FOURTHVALUE);
+        var1.addLast(LIST1FIFTHVALUE);
+        var2.add(LIST2EXTRAVALUE1);
+        var2.add(LIST2EXTRAVALUE3);
+        var2.add(LIST2EXTRAVALUE4);
+
+        try {
+            boolean var3 = LinkedListUtils.containsSubsequence(var1, var2);
+            Assert.assertFalse("containsSubsequence returns true when lists have no overlapping elements",
+                    var3);
+        } catch (Exception var4) {
+            Assert.fail("containsSubsequence throws " + var4 + " when lists have no overlapping "
+                    + "elements");
+        }
+
+    }
+
+    /**
+     * test.
+     */
+    @Test
+    public void testSomeOverlap() {
+        LinkedList var1 = new LinkedList();
+        LinkedList var2 = new LinkedList();
+        var1.add(LIST1FIRSTVALUE);
+        var1.addLast(LIST1SECONDVALUE);
+        var1.addLast(LIST1THIRDVALUE);
+        var1.addLast(LIST1FOURTHVALUE);
+        var1.addLast(LIST1FIFTHVALUE);
+        var2.add(LIST1SECONDVALUE);
+        var2.add(LIST1THIRDVALUE);
+        var2.add(LIST2EXTRAVALUE3);
+
+        try {
+            boolean var3 = LinkedListUtils.containsSubsequence(var1, var2);
+            Assert.assertFalse("containsSubsequence returns true when lists have some overlapping elements "
+                    + "but first does not contain all elements of second", var3);
+        } catch (Exception var4) {
+            Assert.fail("containsSubsequence throws " + var4 + " when lists have some overlapping "
+                    + "elements but first does not contain all elements of second");
+        }
+
+    }
+
+    /**
+     * test.
+     */
+    @Test
+    public void testOverlapWrongOrder() {
+        LinkedList var1 = new LinkedList();
+        LinkedList var2 = new LinkedList();
+        var1.add(LIST1FIRSTVALUE);
+        var1.addLast(LIST1SECONDVALUE);
+        var1.addLast(LIST1THIRDVALUE);
+        var1.addLast(LIST1FOURTHVALUE);
+        var1.addLast(LIST1FIFTHVALUE);
+        var2.add(LIST1FOURTHVALUE);
+        var2.add(LIST1SECONDVALUE);
+        var2.add(LIST1FIFTHVALUE);
+
+        try {
+            boolean var3 = LinkedListUtils.containsSubsequence(var1, var2);
+            Assert.assertFalse("containsSubsequence returns true when first LinkedList contains all elements "
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define a constant instead of duplicating this literal "containsSubsequence returns true when first LinkedList contains all elements " 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)